### DISCUSSION TITLE
Conan recipe for latest release (1.8.1) fails due to ARM architecture.

### DISCUSSION THREAD

**Discussion by rbharvs** [open-telemetry/opentelemetry-cpp#1875] [Q&A] [answered]
Hello, does this group manage the [Conan recipe](https://conan.io/center/opentelemetry-cpp) for this project? When installing on my M1 Macbook Pro, the recipe errs with the following message:

```
opentelemetry-cpp/1.8.1: Invalid ID: opentelemetry-cpp/1.8.1 doesn't support architecture : armv8
```

This is simply due to a hard-coded check in the Conan recipe.

Based on the pull request title from https://github.com/open-telemetry/opentelemetry-cpp/pull/1325 however, it appears that ARM architecture is supported. If so, is it possible for the hard-coded check in the Conan recipe be updated so that I can install the library on my system via the Conan package?

**Comment by lalitb** [comment, marked as answer]
Yes, ARM architecture is supported by otel-cpp, so hardcoding could be removed from Conan package. Would suggest you to create a bug with Conan (or maybe contribute the fix there) as the package is not maintained by OpenTelemetry C++ project. Refer - [Using Package Managers](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md#using-package-managers).

**Reply by rbharvs** [reply]
Thank you for the prompt reply! Sorry I missed that section of the docs before filing an issue.

**Comment by ThomsonTan** [comment]
BTW, opentelemetry-cpp should work on arm64, but currently we don't run any CI on ARM platforms. Please let us know if there is any issue.

https://github.com/open-telemetry/opentelemetry-cpp#supported-development-platforms
